styles/components/profilePic.module.css

```
@@ -8,6 +8,8 @@
   height: 100%;
   overflow-y: auto;
   margin-bottom: 2rem;
+  width: calc(100vw-290px);
```

Biome (1.9.4)

[error] Unexpected unknown unit: vw-290px

See MDN web docs for more details.
Use a known unit instead, such as:


(lint/correctness/noUnknownUnit)